app/pdfs/distribution_pdf.rb

```diff
@@ -19,110 +19,116 @@ def compute_and_render
                    Organization::DIAPER_APP_LOGO
                  end
 
-    image logo_image, fit: [250, 85]
+    footer_height = 35
 
-    bounding_box [bounds.right - 225, bounds.top], width: 225, height: 85 do
-      text @organization.name, align: :right
-      text @organization.address, align: :right
-      text @organization.email, align: :right
-    end
-
-    text "Issued to:", style: :bold
-    font_size 12
-    text @distribution.partner.name
-    move_up 24
+    # Bounding box containing non-footer elements
+    bounding_box [bounds.left, bounds.top], width: bounds.width, height: bounds.height - footer_height do
+      image logo_image, fit: [250, 85]
```

Comment:
All that's happening from lines 26-131 is the original code was indented. Calling this out because this diff looks super busy for something that is a straightforward change.

Reply:
(nods) I checked it in my IDE -- because it's easier to see that kind of change there.    It looks like all of that is  just wrapping a bunch of code in another bounding box.

Reply:
You can also just click "hide whitespace" in the GitHub settings. 

![](https://github.com/rubyforgood/human-essentials/assets/1986893/422c20a3-57d5-4b2d-8d3d-49018e732a30)